builder-support/helpers/update-rust-library-version.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python3
+"""Update the Rust library version in the Cargo.toml file to keep it in sync with the product version."""
+
+import shutil
+import sys
+import tempfile
+
+def main():
+    if len(sys.argv) != 4:
+        print(f'Usage: {sys.argv[0]} <path/to/Cargo.toml> <package name> <version to set>')
+        sys.exit(1)
+
+    file_name = sys.argv[1]
+    package_name = sys.argv[2]
+    version = sys.argv[3]
+
+    with tempfile.NamedTemporaryFile(mode='w+t', encoding='utf-8', delete=False) as generated_fp:
+        with open(file_name, 'r', encoding='utf-8') as cargo_file:
+            in_rust_package_section = False
+            for line in cargo_file:
+                if line.startswith('['):
+                    in_rust_package_section = False
+                elif line == f'name = "{package_name}"\n':
+                    in_rust_package_section = True
+                elif in_rust_package_section and line.startswith("version ="):
+                    generated_fp.write(f"version = \"{version}\"\n")
```

Comment:
A comment in the `.toml` file referring to the meson dist script and saying that the `version =` line is special would be good.

Reply:
Will do.